### ISSUE TITLE
Log level DEBUG does not work

### ISSUE THREAD

**Issue by florianmutter** [freiheit-com/kuberpult#945]
Setting log level to `DEBUG` results in the following error message:

```
failed to IncreaseLevel: invalid increase level, as level "debug" is allowed by increased level, but not by existing core
```

Nothing gets logged. We had the issue that kuberpult-cd-service took longer to start and was killed with the default liveliness settings. This was hard to debug because there is no log message whatsoever.

**Comment by sven-urbanski-freiheit-com** [comment]
We should probably migrate to https://pkg.go.dev/golang.org/x/exp/slog

**Comment by hannesg** [comment]
Yes, this is a know issue. We should definitely do that. Is our exact log format covered by semver stability? If no, we could do this quiet easily.

**Comment by sven-urbanski-freiheit-com** [comment]
I think I would not consider the exact log format as a breaking change.

**Comment by Frank-Reichenbach** [comment]
Still valid for version v2.21.1 (I am aware that this is not the current version)
